.github/workflows/pull_request.yml

```diff
@@ -115,3 +116,9 @@ jobs:
             repo: context.repo.repo,
             body: `The ${process.env.JAR_VERSION} JAR file for this MR can be found here: ${process.env.JAR_URL}`
           })
+    
+    - name: Publish with Gradle
```

Comment:
If we go with publishing to packages, I don't need to create releases for PRs anymore

Reply:
Clean up tasks: 

- [ ] Add beta to package version
- [ ] Add publishing to `main` workflow
- [ ] Cleanup things not needed (creating releases for PRs)